Common/src/CConfig.cpp

```diff
@@ -7200,6 +8139,10 @@ bool CConfig::GetSolid_Wall(unsigned short iMarker) const {
 
   return (Marker_All_KindBC[iMarker] == HEAT_FLUX  ||
           Marker_All_KindBC[iMarker] == ISOTHERMAL ||
+          Marker_All_KindBC[iMarker] == HEAT_FLUX_NONCATALYTIC  ||
+          Marker_All_KindBC[iMarker] == ISOTHERMAL_NONCATALYTIC ||
+          Marker_All_KindBC[iMarker] == HEAT_FLUX_CATALYTIC  ||
+          Marker_All_KindBC[iMarker] == ISOTHERMAL_CATALYTIC ||
```

Comment:
If you could re-use the normal HEAT_FLUX and ISOTHERMAL enums these changes could be avoided, and this type of change is the main source of bugs in SU2, invariably some area is forgotten or someone new comes along and does not know the extra markers exist... So please please consider re-using everything you can, please :)

Reply:
I am not sure if I understand the suggestion @pcarruscag .  HEAT_FLUX_NONCATALYTIC , ISOTHERMAL_NONCATALYTIC , HEAT_FLUX_CATALYTIC, ISOTHERMAL_CATALYTIC are bcs different than the normal HEAT_FLUX and ISOTHERMAL, therefore need different flags in the cfg file and corresponding options in option structure, to be used somewhere else.

Reply:
Ok, I thought these were small variations on the normal heatflux / isothermal boundaries (atm non-catalytic just calls the normal implementation and catalytic makes use of the normal version right?), and that it might be more intuitive for an user to specify those properties separately.
By having different boundary condition names for those, you make them incompatible with new features added to the code, for example #877. 
I do not have domain knowledge about your work, I am just trying to have the multi physics nature of SU2 in mind, the same way we have abstractions for solvers, we also need to have abstractions for options, otherwise we have a combinatoric explosion every time something new is added...